src/state/shellscripts/bwrap.sh

```diff
@@ -16,43 +11,50 @@ fi
 
 # --new-session requires bubblewrap 0.1.7
 # --die-with-parent requires bubblewrap 0.1.8
-ARGS=(--unshare-net --new-session --die-with-parent)
-ARGS=("${ARGS[@]}" --proc /proc --dev /dev)
-ARGS=("${ARGS[@]}" --setenv TMPDIR /opam-tmp --setenv TMP /opam-tmp --setenv TEMPDIR /opam-tmp --setenv TEMP /opam-tmp)
-ARGS=("${ARGS[@]}" --tmpfs /opam-tmp)
-ARGS=("${ARGS[@]}" --tmpfs /run)
+ARGS="--unshare-net --new-session --die-with-parent"
+ARGS="$ARGS --proc /proc --dev /dev"
+ARGS="$ARGS --setenv TMPDIR /opam-tmp --setenv TMP /opam-tmp --setenv TEMPDIR /opam-tmp --setenv TEMP /opam-tmp"
+ARGS="$ARGS --tmpfs /opam-tmp"
+ARGS="$ARGS --tmpfs /run"
 # NOTE: When adding a new mount-point please sync with the loop below to avoid overriding the mount point
 
+#$1 is permission, $2 is host dir, $3 is sandbox dir
 add_mount() {
-    if [ -d "$dir" ]; then
-      case "$1" in
-          ro) B="--ro-bind";;
-          rw) B="--bind";;
-          sym) B="--symlink";;
-      esac
-      ARGS=("${ARGS[@]}" "$B" "$2" "$3")
-    fi
+  if [ -d "$2" ]; then
+          case $2 in
+                  ro) B="--ro-bind";;
+                  rw) B="--bind";;
+                  sym) B="--symlink";;
+          esac
+          ARGS="$ARGS $B $2 $3"
+  fi
 }
 
+#$1 is permission, rest are directories mapped 1-1 from host to sandbox
 add_mounts() {
-    local flag="$1"; shift
```

Comment:
`local` is a valid POSIX sh construct. Could you keep it?

Reply:
`local` isn't posix yet. (https://github.com/emersion/mrsh/issues/183). That said, its probably supported enough that keeping it wouldn't hurt.